future/name-resolution/002-nr-host-synchronization/README.md

```diff
@@ -59,7 +63,11 @@ avahi         15  0.0  0.0   7136  1300 ?        S    18:14   0:00 avahi-daemon:
 root          42  0.0  0.0   3472  1792 pts/0    S+   18:20   0:00 grep --color=auto avahi
 ```
 
-We can see that the `avahi-daemon` is running. Huzzah! We should be able to use it to perform local name resolution. Let's start by `ping`ing `host-f`. The first thing `ping` will have to do is resolve the hostname to an IP address. But, just like in other chapters, we wanna see what's happening in all this communication. Let's use our old friend `tcpdump`!
+We can see that the `avahi-daemon` is running. Huzzah!
+
+We should be able to that running service to perform name resolution on our local network. Let's start by `ping`ing `host-f`. The first thing `ping` will have to do is resolve the hostname to an IP address. But, just like in other chapters, we wanna see what's happening in all this communication. Let's use our old friend `tcpdump`!
```

Comment:
```suggestion
We should be able to use that running service to perform name resolution on our local network. Let's start by `ping`ing `host-f`. The first thing `ping` will have to do is resolve the hostname to an IP address. But, just like in other chapters, we wanna see what's happening in all this communication. Let's use our old friend `tcpdump`!
```

Reply:
this also seems like a great place to add that disclaimer that we're using `ping`, but a more traditional way of looking this up would be `dig`. I forget what the problem was with using `dig` though? I think `dig` didn't follow the `/etc/resolv.conf` hosts sequence?